Changelog.md

```diff
@@ -1,3 +1,8 @@
+# 1.1.0 - 2016-04-28
+
+- Upgrade Minim to 0.14.0 and thus remove the short hand notation in fury
```

Comment:
Yeah. I have no issues doing v2 now but we will be adding the compact refract back anyway, which is why I did v1.1. I will leave the decision upto you.